src/_P067_HX711_Load_Cell.ino

```diff
@@ -28,8 +28,8 @@
 #define PIN(n) (Settings.TaskDevicePin[n][event->TaskIndex])
 #endif
 
-int32_t Plugin_067_OversamplingValue = 0;
-int16_t Plugin_067_OversamplingCount = 0;
+int32_t Plugin_067_OversamplingValue[TASKS_MAX];
+int16_t Plugin_067_OversamplingCount[TASKS_MAX];

```

Comment:
Please change these to use 
- `std::map<byte, int32_t> Plugin_067_OversamplingValue;` 
- `std::map<byte, int16_t> Plugin_067_OversamplingCount;` 

Also there must be a PLUGIN_EXIT part:

```C++
    case PLUGIN_EXIT:
      {
        Plugin_067_OversamplingValue.erase(event->TaskIndex);
        Plugin_067_OversamplingCount.erase(event->TaskIndex);
        break;
      }
```